umh.docs.umh.app/content/en/docs/production-guide/installation/flatcar-installation-bare-metal.md

```diff
@@ -13,106 +13,162 @@ aliases:
 
 Here is a step-by-step guide on how to deploy the UMH stack on
 [Flatcar Linux](https://www.flatcar.org/), a Linux distribution designed for
-container workloads, with high security and low maintenance.
+container workloads with high security and low maintenance. This will leverage the UMH Device and Container Infrastructure, which is used in this example as the community edition. During the installation, we will download and boot on your device / on our VM from the bootloader iPXE. iPXE, if downloaded further below, is pre-configured and tailored to the UMH. First, it will guide you through an installation assistant. Then, it will load flatcar from the latest LTS version (current-2023) and boot into it. This system is called flatcar-0. Then it will install flatcar to disk. Upon automatic rebooting, it will boot into the installed flatcar instance called flatcar-1, which will automatically install the UMH.
 
-This is a good option if you want to deploy the UMH stack on edge devices or IPCs.
+## Prerequisites
 
-## {{% heading "prerequisites" %}}
+Before initiating the deployment process, ensure your system aligns with the following prerequisites:
 
-{{< include "task-befinst-prereqs.md" >}}
-
-You need the latest version of our iPXE boot image:
+CPU Cores: Minimum of 4
+Memory: At least 8 GB
+Storage: Minimum 32 GB disk space
+iPXE Boot Image: Obtain the latest version appropriate for your system architecture:
 
 - [ipxe-x86_64-efi](https://github.com/united-manufacturing-hub/ipxe/releases/latest/download/ipxe-x86_64-efi.usb):
-  for reasonably modern systems.
+  Suitable for modern systems, recommended for Proxmox.
 - [ipxe-x86_64-bios](https://github.com/united-manufacturing-hub/ipxe/releases/latest/download/ipxe-x86_64-bios.usb):
-  for older systems.
+   For legacy systems.
 - [ipxe-arm64-efi](https://github.com/united-manufacturing-hub/ipxe/releases/latest/download/ipxe-arm64-efi.usb):
-  for ARM systems. Currently, Raspberry Pi 4 is **not** supported.
+  For ARM architectures (**Note**: Raspberry Pi 4 is currently unsupported).
+
+
+### Deployment Overview
+## Booting from the iPXE image
+
+### Bare Metal Installation
+1. **USB Preparation**: Write the iPXE image to a USB stick. Refer to our guide on [Flashing Operating Systems to USB]((https://learn.umh.app/course/flashing-an-operating-system-onto-a-usb-stick/)) for detailed instructions.
+
+2. **SSH Client Requirement**: Ensure your computer has an SSH client installed.
+3. Network Configuration: Setup should include:
+- Internal Network
+- WAN and LAN connections
+- Edge Device
+- Router Setup
+
+4. IP Addressing: Assign a static IP address to your edge device for consistent network performance. Configure this either via a static lease in your DHCP server or during the installation process. Avoid changing the IP post-installation to prevent certificate issues.
+
+### Virtual Machine Setup
+1. **VM Creation**: In your virtualization software, create a new VM with these specifications:
```

Comment:
One problem we had in the oast that we wrote too often the requirements in various places, but then updated it only in a few. Maybe for CPU, Memory, Hard Disk and others (potentially also SSE4.2), refer to a different article?